docs/faq.rst

```diff
@@ -207,3 +207,55 @@ Will KSQL work with a HTTPS Confluent Schema Registry?
 
 Yes. KSQL can be configured to communicate with the Confluent Schema Registry over HTTPS. For more information, see
 :ref:`config-security-ksql-sr`.
+
+================================================
+Where are KSQL-related data and metadata stored?
+================================================
+
+Metadata is stored in and built from the KSQL command topic. Each KSQL server
+has its own in-memory version of the metastore. To secure the metadata, you must
+secure the command topic.
+
```

Comment:
should we use command syntax for the KSQL statements? i could see an argument either way, up to you.